constraint-solver/src/indexed_constraint_system.rs

```diff
@@ -186,6 +186,10 @@ impl<T: RuntimeConstant, V: Clone + Eq> IndexedConstraintSystem<T, V> {
             ConstraintSystemItem::BusInteraction,
         );
     }
+
+    pub fn into_system(self) -> ConstraintSystem<T, V> {
```

Comment:
`From<IndexedConstraintSystem<T, V>>` is implemented for `ConstraintSystem<T, V>`